packages/react-components/src/DocsComponents/FluentDocsDocsStory.stories.tsx

```diff
@@ -0,0 +1,168 @@
+  if (codesandboxForm) {
+    const codesandboxFormData = new FormData(codesandboxForm);
+    codesandboxFormData.append('json', '1');
+    const codesandboxResponse = await fetch(codesandboxForm.action, {
+      method: 'post',
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      body: new URLSearchParams(codesandboxFormData),
+    });
+
+    const sandboxId = (await codesandboxResponse.json()).sandbox_id;
+    codesandboxUrl = `https://codesandbox.io/s/${sandboxId}?file=/example.tsx`;
+  }
+
+  // open GitHub
+  const newIssueUrl = newGithubIssueUrl({
+    repoUrl: packageJson.repository.url,
+    title: '',
+    labels: ['Type: Bug :bug:', 'Fluent UI vNext', componentName ? `Component: ${componentName}` : ''],
+    template: `bug_report_v9_${componentName}_permit.md`,
+    body: bugReportTemplate
+      .replace('[package-versions]', packageJson.version)
+      .replace('[browser-and-os-versions]', navigator.userAgent)
+      .replace('[link-to-codesandbox]', codesandboxUrl),
+  });
+  window.open(newIssueUrl, '_blank');
+};
+
+const LivePreview: React.FunctionComponent<DocsStoryProps & { componentName: string }> = ({ id, componentName }) => {
+  const [renderInline, setRenderInline] = React.useState(true);
+  const sandpack = useSandpack();
+
+  React.useEffect(() => {
+    if (id) {
+      const anchor = document.getElementById(`anchor--${id}`);
+      const iframe = anchor?.getElementsByTagName('iframe')?.item(0);
+
+      // toggle iframe/inline rendering
+      if (iframe) {
+        iframe.style.display = 'none';
+
+        sandpack.listen(msg => {
+          if (msg.type === 'start') {
+            iframe.style.display = 'block';
+            setRenderInline(false);
+          }
+        });
+      }
+    }
+  }, [id]);
+
+  return (
+    <SandpackPreview
+      actionsChildren={
+        <>
+          <button
+            type="button"
+            className="sp-button"
+            style={{ padding: 'var(--sp-space-1) var(--sp-space-3)' }}
+            onClick={handleReportBug(componentName)}
+          >
+            Report bug
+          </button>
+        </>
+      }
+    >
+      <>{renderInline && <Story id={id} />}</>
+    </SandpackPreview>
+  );
+};
+
+// Most of this file is copied from Storybook's addons/docs/src/blocks/DocsStory.tsx
+export const FluentDocsDocsStory: React.FunctionComponent<DocsStoryProps & { componentName: string }> = ({
+  id,
+  name,
+  expanded = true,
+  withToolbar = false,
+  parameters = {},
+  componentName,
+}) => {
+  const { docs } = parameters;
+
+  if (!name) {
+    return null;
+  }
+
+  return (
+    <Anchor storyId={id ?? ''}>
+      {expanded && name && <Subheading>{name}</Subheading>}
+      {expanded && docs && docs.description?.story && <Description markdown={docs.description?.story} />}
+      <div className="sbdocs sbdocs-preview">
+        <SandpackProvider
+          css={{ width: 'fit-content' }}
+          template="react"
```

Comment:
```suggestion
          template="react-ts"
```

Should it be `react-ts`?
https://sandpack.codesandbox.io/docs/api/react/#sandpackpredefinedtemplate